data/human/news.txt

```diff
@@ -1158,3 +1158,31 @@ news "human farming in pirate space"
 			"Beneath the exterior of many pirates, there's a person who desperately wishes that they'd never taken that path and could return to living normally. I'm talking from personal experience."
 		word
 			`"`
+
+
+
+news "grafiti on spaceport wall"
```

Comment:
```suggestion
news "graffiti on spaceport wall"
```
I know this will never show up in-game, but having correct spelling will definitely make it easier to find in the file for future additions.